app/assets/javascripts/discourse/tests/acceptance/composer-editor-mentions-test.js

```diff
@@ -65,11 +61,14 @@ acceptance("Composer - editor mentions", function (needs) {
     await visit("/");
     await click("#create-topic");
 
-    await emulateAutocomplete(".d-editor-input", "abc @u");
-    await click(".autocomplete.ac-user .selected");
+    const editor = query(".d-editor-input");
+
+    for (let c of "abc @u\r") {
+      await simulateKey(editor, c);
```

Comment:
Replaced the hacky `emulateAutocomplete` with `simulateKey`.

Also replaced the `await click(".autocomplete.ac-user .selected");` with emulating pressing <kbd>Enter</kbd> (aka "\r") so it's (hopefully) more reliable and faster.